extensions/amp-timeago/1.0/test-e2e/test-bento-timeago.js

```diff
@@ -0,0 +1,44 @@
+describes.endtoend(
+  'bento-timeago',
+  {
+    version: '1.0',
+    fixture: 'bento/bento-timeago.html',
+    experiments: ['bento-timeago'],
```

Comment:
You should be able to remove this line—experiments are only defined and enforced in AMP mode.